packages/webapp/src/components/Task/AddProduct/index.jsx

```diff
@@ -107,6 +107,7 @@ const AddProduct = ({
       <input name={NAME} style={{ display: 'none' }} {...register(NAME, { required: true })} />
       <input name={PRODUCT_ID} style={{ display: 'none' }} {...register(PRODUCT_ID)} />
       <Input
+        data-cy={'addTask-supplier'}
```

Comment:
Should this just be a string?

Reply:
it's used as uid in cypress tests. Webstorm always auto-complete bracket